samples/dfu/README.rst

```diff
@@ -7,44 +7,70 @@ Overview
+built with a new version number, registered as an artifact, and depolyed as a
+release. The device will detect this change and automatically download and
+install.
+
 Requirements
 ************
 
 - Golioth credentials
 - Network connectivity
 
-Using nRF9160 Feather with nRF Connect SDK
-******************************************
+Using nRF9160 Feather or Thingy:91 with nRF Connect SDK
+*******************************************************
+
+A full nRF9160 firmware `flashing overview`_ is available in the Golioth docs.
+Please use that resource to install the tools necessary to flash these devices.
+
+Configure the following Kconfig options based on your Golioth credentials:
+
+- GOLIOTH_SYSTEM_CLIENT_PSK_ID  - PSK ID of registered device
+- GOLIOTH_SYSTEM_CLIENT_PSK     - PSK of registered device
 
-Build Zephyr sample application for nRF9160 Feather:
+by adding these lines to configuration file (e.g. ``prj.conf``):
+
+.. code-block:: cfg
+
+   CONFIG_GOLIOTH_SYSTEM_CLIENT_PSK_ID="my-psk-id"
+   CONFIG_GOLIOTH_SYSTEM_CLIENT_PSK="my-psk"
+
+Build Zephyr sample application:
 
 .. code-block:: console
 
-   $ west build -b circuitdojo_feather_nrf9160_ns samples/dfu
+   $ #Build for the Circuit Dojo nrf9160 Feather:
+   $ west build -b circuitdojo_feather_nrf9160_ns samples/dfu -p
+   $ #or build for the Thingy:91:
+   $ west build -b thingy91_nrf9160_ns samples/dfu -p
 
-Enter bootloader and use ``mcumgr`` to flash firmware:
+Enter bootloader mode and use ``mcumgr`` to flash firmware:
 
 .. code-block:: console
 
-   $ mcumgr --conntype serial --connstring /dev/ttyUSB,baudrate=1000000 build/zephyr/app_update.bin
+   $ #Flashing the Circuit Dojo nRF9160 Feather
+   $ newtmgr --conntype=serial --connstring='dev=/dev/ttyUSB0,baud=1000000' image upload build/zephyr/app_update.bin
+   $ #Flashing example for Thingy:91
+   $ newtmgr --conntype=serial --connstring='dev=/dev/ttyACM0,baud=115200' image upload build/zephyr/app_update.bin
 
-Now rebuild application with assigned new version to 1.2.3 to distinguish it
-from old firmware:
+Now rebuild the application with the new version number 1.2.3 to distinguish it
+from the old firmware:
 
 .. code-block:: console
 
    $ west build -p -b circuitdojo_feather_nrf9160_ns samples/dfu -- -DCONFIG_MCUBOOT_IMAGE_VERSION=\"1.2.3\"
 
-Follow `Start DFU using goliothctl`_ to create new firmware release and roll it
-out to device, but use ``build/zephyr/app_update.bin`` instead of ``new.bin`` in
-the first step when uploading new artifact:
+Jump down to the `Start DFU using goliothctl`_ section lower on this page and
+follow those steps to create a new firmware release and roll it out to the
+device. **IMPORTANT:** use ``build/zephyr/app_update.bin`` instead of
+``new.bin`` in the first step when uploading the new artifact:
 
 .. code-block:: console
 
    $ goliothctl artifact create build/zephyr/app_update.bin --version 1.2.3
 
-See `nRF9160 Feather Programming and Debugging`_ for details.
-
```

Comment:
Why don't we want to reference official documentation? Would be great to write down that reason in commit description.